docs/install/Knative-custom-install.md

```diff
@@ -343,11 +343,17 @@ commands below.
      `[COMPONENT]`, `[VERSION]`, and `[FILENAME]` are the Knative component,
      release version, and filename of the Knative component or plugin. Examples:
 
-     - `https://github.com/knative/serving/releases/download/v0.5.2/serving.yaml`
+     - `https://github.com/knative/serving/releases/download/v0.5.2/serving.yaml --selector networking.knative.dev/certificate-provider!=cert-manager`
      - `https://github.com/knative/build/releases/download/v0.5.0/build.yaml`
      - `https://github.com/knative/eventing/releases/download/v0.5.0/release.yaml`
      - `https://github.com/knative/eventing-sources/releases/download/v0.5.0/eventing-sources.yaml`
 
+     **Note**: By default, the Knative Serving component installation (`serving.yaml`) includes a 
```

Comment:
@ZhiminXiang here is my edited version (that ive re-positioned up in the text), PTAL

Reply:
LGTM